### PR TITLE
Use own eslint file

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,14 +3,11 @@
   "version": "0.3.0",
   "description": "Geoblocks projections",
   "scripts": {
-    "eslint": "eslint --config=node_modules/@geoblocks/base/.eslintrc.yml src/*.js",
-    "eslint-fix": "eslint -config=node_modules/@geoblocks/base/.eslintrc.yml src/*.js --fix",
-    "build": "webpack --mode production",
-    "build-debug": "webpack --mode development",
-    "start": "webpack-dev-server --mode development --content-base dist/ --watch",
```
[the remark this answers is not in the document]
Comment (gberaudo) [reply package.json]: We have no example in this block. These targets are currently failing.

File: package.json [hunk 2]
```diff
@@ -3,14 +3,11 @@
...
+    "eslint": "eslint src/*.js",
+    "eslint-fix": "eslint src/*.js --fix",
```
[the remark this answers is not in the document]
Comment (gberaudo) [reply package.json]: Yes, it is rare to call eslint --fix anyway so having a target for it is not necessary IMO.

**Comment by gberaudo** [comment]
@sbrunner, this change makes the configuration known by editors. @fredj can show you a demo.

**Comment by sbrunner** [comment]
If it's that We should rename the npm package `@geoblocks/base` to e.-g. `geoblocks-base` to be able to do standard eslint links ...

File: .travis.yml [hunk 3]
```diff
@@ -8,8 +8,3 @@ node_js:
 cache:
   directories:
     - node_modules
-
-script:
-  - npm run lint
-#  - npm run test
-  - npm run doc
```
Comment (sbrunner) [inline .travis.yml]: Why remove the scripts, you want to remove Travis?

Comment (gberaudo) [reply .travis.yml]: No it is running with travis default (npm test).

Comment (sbrunner) [reply .travis.yml]: OK, thanks :-)

**Comment by gberaudo** [comment]
The base .eslintrc.yml is very thin so it is not a burden to maintain a copy in each block.
Moreover the base rules are debatable:
- should console statement be allowed?
- do we want comma dangle?
- is it OK to accept invalid jsdoc?

It is better if the block authors answer these questions themselves.

**Pull request merged**